sechub-wrapper-xray/src/test/java/com/mercedesbenz/sechub/wrapper/xray/report/XrayWrapperReportReaderTest.java

```diff
@@ -24,24 +26,24 @@ class XrayWrapperReportReaderTest {
     @BeforeEach
     void beforeEach() {
         reportReader = new XrayWrapperReportReader();
-        mapper = new ObjectMapper();
+        mapper = XrayWrapperJSONConverter.get().getMapper();
     }
 
     @Test
-    void getReportFiles_get_valid_reports() {
+    void getReportFiles_get_valid_reports() throws XrayWrapperReportException {
         /* prepare */
         String source = "src/test/resources/xray-report-examples";
         String resultFile = "resultfile";
         String cycloneReport = "Docker_SBOM_Export_CycloneDX.json";
         String securityReport = "Docker_Security_Export.json";
 
         /* execute */
-        reportReader.getReportFiles(source, resultFile);
+        reportReader.findXrayReportsInArchive(source, resultFile);
```

Comment:
this is a find method... normally I would expect something like this.

``` java
finder = new MyFinder();

MyFinderResult result = finder. findXyz(params)

assertEquals("expected", result.getCycloneName())
// etc. etc.
```

Your current implementation is stateful and would never be thread safe.
It is also difficult to understand why a find... does not return the find result. Means: "find... what?"